src/schema/posts.ts

```diff
@@ -1783,60 +1783,58 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
       _,
       { id }: { id: string },
       ctx: Context,
-    ): Promise<GQLPostSmartTitle> => {
-      const post: Pick<Post, 'title' | 'contentMeta'> = await queryReadReplica(
-        ctx.con,
-        ({ queryRunner }) =>
-          queryRunner.manager.getRepository(Post).findOneOrFail({
+    ): Promise<GQLPostSmartTitle> =>
+      queryReadReplica(ctx.con, async ({ queryRunner }) => {
```

Comment:
Forgot to this in my previous PR

Reply:
Only change is RR?

Reply:
Yeah

Reply:
Guess looking at diff is difficult because of indentation and prettier doing things :triggered: